src/components/Header/index.jsx

```diff
@@ -0,0 +1,18 @@
+const Header = ({invoicesjson}) => {
+    const unpaid = invoicesjson.filter(findUnpaid);
+
+    function findUnpaid(invoice) {
```

Comment:
This function is short enough that I would just use it as a fat-arrow function inside of `invoicesjson.filter`